### ISSUE TITLE
Communicate within page that the MHV landing page is authenticated (MVP)

### ISSUE THREAD

**Issue by wesrowe** [department-of-veterans-affairs/va.gov-team#68424]
### User story

As an authenticated user on the MHV Landing Page, I want to see within the main landing page that I am on a personalized, logged-in page, so that I can immediately recognize my logged-in status and navigate the landing page with confidence and trust.

[Document with personalization ideas](https://docs.google.com/document/d/1MbPK1MYjbDrmOOnnru7eD0O-XYweEp3KPDOJntuRnkA/edit)

## Notes

- Veterans stated in recent research that they did not realize the landing page has information unique to them
- If use of a new endpoint is needed to support a solution, we should coordinate with the team owning the API for visibility.
- Consider existing personalization on MHV National Portal and My VA
- Consider displaying priority group for users with healthcare
- [Slack thread](https://dsva.slack.com/archives/C0581MN69TJ/p1698951750274489?thread_ts=1698948836.736099&cid=C0581MN69TJ) about MHV help desk that may be relevant to future IA
- Consider how this will/can extend more broadly across the VAgov ecosystem

## Acceptance criteria

- Determined whether we want a single personalization solution with conditional elements, or to create multiple designs to cover different user types (e.g., consider whether user has health care, does not have health care, facility registration status, etc.)
- A personalized experience is presented to _some or all_ authenticated users
- Documented UX gaps and opportunities that emerge from the discussion of personalization options

## Tasks

- [x] Add html attribute(s) to [tell Datadog](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/analytics/setup-real-user-monitoring.md#privacy) that the user's name should not be collected
- [ ] Ticketed any further discovery story(s) that are identified as useful to iterate on this story's outcomes [team+Wes]
- [x] Implement designs behind feature flag.
- [x] Staging accessibility review of feature

**Comment by MikeC-A6** [comment]
@wesrowe story revision for consideration:

As an authenticated user on the MHV Landing Page, I want to see a distinct indicator(s) that I am on a personalized, logged in page so that I can immediately recognize my logged-in status and navigate the landing page with confidence.

**Comment by wesrowe** [comment]
@MikeC-A6 – edited, but can you clarify something. in another story you commented that including a hint toward personalization is putting a thumb on the scale for solutioning. however, in this story you added it. is this because the team has already identified "personalization" as the solution approach?

**Comment by wesrowe** [comment]
Related priority group tickets: #61045 (UX), #61049 (implementation).

**Comment by MikeC-A6** [comment]
For discussion in refinement:  
As an authenticated user **with VA healthcare visiting** the MHV Landing Page..."

**Comment by radavis** [comment]
Work regarding Priority Groups: https://github.com/department-of-veterans-affairs/vets-website/pull/25343

**Comment by radavis** [comment]
Work implementing Priority Groups: https://github.com/department-of-veterans-affairs/vets-website/pull/25343

Note, we need to coordinate with the 1010-ez team before we start making requests to the API endpoint that gives us HCA status and Priority Group.

Another possible way to indicate that the user is logged in and the page is "personalized" would be to use the NameTag component:
![Screenshot 2023-10-30 142045](https://github.com/department-of-veterans-affairs/va.gov-team/assets/1299034/51c251a6-950c-4bf4-8648-064472acd245)

**Comment by fmccaf1** [comment]
MVP design - waiting on feedback from team

![Priority group w learn more link Copy 33](https://github.com/department-of-veterans-affairs/va.gov-team/assets/106624475/86e4eac0-c801-415f-b29d-829b7a66a9e6)

**Comment by radavis** [comment]
Preliminary designs (pulled from Slack):

![Personalization MVP - mobile](https://github.com/department-of-veterans-affairs/va.gov-team/assets/1299034/abff68fd-2e0a-4f6b-a91e-be80a7fe4631)

![Personalization MVP - desktop](https://github.com/department-of-veterans-affairs/va.gov-team/assets/1299034/85bd77cd-fdc4-4321-9af4-2af4b39d58bb)

@fmccaf1 I'm wondering if I should feature toggle this entire change (copy change plus 'Welcome, Jesse"/Profile link), or only feature toggle the "Welcome, Jesse"/Profile link. Thoughts?

**Comment by fmccaf1** [comment]
Answered in Slack thread 11/21: 

> Good question, I think we're wanting to feature flag the whole thing (including content changes). That was definitely the sense I got from Mike, that this was the plan he was envisioning. If you disagree let me know!


Here are screens with changes to white space above H1:

**Comment by wesrowe** [comment]
@radavis – I just realized that the user's first name is PII [we want to hide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/analytics/setup-real-user-monitoring.md#privacy) in DD session replays. Did you already take care of that?

**Comment by wesrowe** [comment]
also note – today Patrick asked that we not release this until it has been reviewed at the Wednesday design review meeting.

**Comment by radavis** [comment]
@wesrowe, Good call on masking the user's name. It's taken care of, now: https://github.com/department-of-veterans-affairs/vets-website/pull/26790/commits/3d06e9ad2a70d39f9f538341ba6252898e497bca

**Comment by fmccaf1** [comment]
Has been reviewed at Wednesday design review, has been discussed with Profile team. From my pov, this ticket is done on the design side of things

**Comment by fmccaf1** [comment]
For the sake of documentation:
We're working on trying to stay in sync with other dependencies with decision to use Veteran's first name or preferred name for LP. 
- Auth dropdown on the site uses first name
- There's also name in authenticated experience's name tag banner (first name also?)

**Comment by wesrowe** [comment]
Update: I scheduled a slack message for Samara and AJ on Monday morning.

**Comment by wesrowe** [comment]
### Notes from quick meeting on the path forward after inputs from various VA design leads

Decisions:
- We will release the design as-is, leveraging preferred name when available, then falling back to first name, and then email(??)

Notes on using data to validate this change:
- The interactive element that we can measure in GA is the Profile link beside the greeting

Hypotheses: If we add the "Hi, Jesse" design with a profile link next to it...
- ... clicks on the Profile link in the header's authenticated menu will be reduced by 20% (ie cannibalized by the new in-page link)
- ... 10% more landing page users will visit the Profile page in aggregate across the two Profile links

Other meeting notes:
- We want to start visually subtle, increase visibility later if we don't see signal
- A11y is a key reason we want to keep the new component left-aligned

**Comment by wesrowe** [comment]
@fmccaf1 @sterkenburgsara @jonathanjnelson @AngelaFowler82 @keithcheung828 – we didn't discuss the fallback when first name is not available. Travis (Profile PM) said he has data that the no-first-name use case numbers are <1000 across MHV and Cerner/Oracle. Are you okay with using the same fallback as the top-right authenticated menu, which is email?

**Comment by fmccaf1** [comment]
Thanks for writing these up and for the question @wesrowe. I still think it'd be less clunky to simply omit first name (and comma) and have the saluation be "Welcome". I'm thinking that In the absence of preferred name and first name, anything else calls attention that we don't know their name. Would love to hear other teammates' thoughts about this.

**Comment by fmccaf1** [comment]
I'm wondering also if the color of the horizontal rule under "Welcome Jesse" could be darkened slightly. Not the lightest gray, but a step up if possible? @radavis @jzucadi

**Comment by jzucadi** [comment]
@fmccaf1 yes, that's no problem.

**Comment by fmccaf1** [comment]
Interesting idea with the blue @jzucadi. I love that you made that suggestion so feel free to do that in the future! I'm thinking no. 2, the two levels up option. Thanks for prompting me to respond here @radavis

**Comment by wesrowe** [comment]
@AngelaFowler82, did you ever check out the final version in Staging?

Also looking to @fmccaf1 for design review.

**Comment by wesrowe** [comment]
@radavis, I tried testing the fallback order and didn't get the preferred name to show up.

Steps I took:
- log in as va.my.health user 10
- see "Welcome, Charlene"
<img width="322" alt="image" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/4054752/712e6604-c23f-4265-a36c-8074323d678e">
- go to profile, where I see that Charlene doesn't have a preferred name yet
- <img width="324" alt="image" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/4054752/d9065e69-d880-4559-81dc-772b645b4d12">
- enter "Romulus" as preferred name
- <img width="313" alt="image" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/4054752/2482f12a-5611-4b4a-aa31-3aba2d52fb2d">
- See that landing page still calls me Charlene, despite hard refresh and then logging out and in again
